### ISSUE TITLE
catch errors in taf.bootstrap source={script}

### ISSUE THREAD

**Issue by colinpmillar** [ices-tools-prod/icesTAF#8]
if a script errors when calling `taf.bootstrap()` with a `DATA.bib` entry that uses a script.  Then the state of the working directory is left in `/bootstrap/data/'.

perhaps replace:
https://github.com/ices-tools-prod/icesTAF/blob/a4841b398159d6ae59350770e84acfd8fd984325/R/process.inner.R#L47-L50

with
```r
    owd <- setwd(dir)
    on.exit({
      setwd(owd)
    })
    source(script)
    setwd(owd)
```

**Comment by colinpmillar** [comment]
Current quick fix:
```r
library(<package>, lib.loc = "../../library")
```

**Comment by arnima-github** [comment]
Well spotted, fixed in 60d556b.
